src/views/MangaPreviewView.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import mangaComponent from "../components/MangaComponent.vue"
+import mangaStampComponent from '../components/MangaStampComponent.vue'
+
+let page = ref(0);
+const props = defineProps({
+  mangaId: {
+    type: String, required: true
+  }
+})
```

Comment:
```suggestion
interface Props{
  mangaId: string
}
const props = defineProps<Props>()
```
の方がTypeScriptらしい書き方になります
requiredでないときは

```suggestion
interface Props{
  mangaId?: string
}
const props = withDefauts(defineProps<Props>() ,{ manageId: "") 
```
みたいにして`?`をつけるとoptionalにできて、`withDefaults`でデフォルト値も指定できます